indico/modules/events/reminders/templates/emails/event_reminder.txt

```diff
@@ -7,16 +7,25 @@
 
 
 {% block subject -%}
-    [{% trans %}Event reminder{% endtrans %}] {{ event.title }} ({{ event.start_dt | format_datetime('short', timezone=event.tzinfo) }} {{ event.timezone }})
+    [{% trans %}Event reminder{% endtrans %}] {{ event.title }}
+    {%- if event.label %} {{ '{{{}}}'.format(event.label.title) }} {% endif -%}
+    ({{ event.start_dt | format_datetime('short', timezone=event.tzinfo) }} {{ event.timezone }})
```

Comment:
I think you lose the space before the `(` when there's no label. Unfortunately the subject is not covered by the snapshots... might be worth at some point to refactor these snapshots so they also include the subject somehow...